source/server/listener_impl.h

```diff
@@ -494,6 +499,7 @@ class ListenerImpl final : public Network::ListenerConfig,
 
   Quic::QuicStatNames& quic_stat_names_;
   MissingListenerConfigStats missing_listener_config_stats_;
+  uint32_t concurrency_;
```

Comment:
nit: const. As an aside it would be nice to not have to store this in every listener, but there probably isn't a great way to do that without passing through all server options so I guess it's ok.

Reply:
I think I can get that from the server instance directly, let me try, also thanks point this out, it indeed better to not pass through it, and it doesn't feel like something the `ListenerImpl` need to hold one.